application_handlers.go

```diff
@@ -89,12 +89,17 @@ func ApplicationCreate(c echo.Context) error {
 		return err
 	}
 
+	sourceDao, err := getSourceDao(c)
+	if err != nil {
+		return err
+	}
+
 	input := &m.ApplicationCreateRequest{}
 	if err = c.Bind(input); err != nil {
 		return err
 	}
 
-	err = service.ValidateApplicationCreateRequest(input)
+	err = service.ValidateApplicationCreateRequest(sourceDao, input)
```

Comment:
Rather than pass a DAO - Should we just pass the tenant ID or request params? That way in the sub-function we can get whatever DAO's we would like.

Reply:
ok, changed